docs/installation/source.md

```diff
@@ -3,43 +3,42 @@
 -->
 # Installation From Source
 
-In order to run imap-backup from source, you'll need Ruby and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+In order to run imap-backup from source, you'll need [Ruby](https://www.ruby-lang.org/en/documentation/installation/) and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
 
-Next, clone the repository
+Next, clone the repository:
 
 ```sh
 git clone https://github.com/joeyates/imap-backup.git
-cd imap-backup
 ```
 
-If you want to use a branch other than `main`
+If you want to use a branch other than `main`:
 
 ```sh
 git checkout --track -b BRANCH origin/BRANCH
 ```
 
-Install dependencies
+Install dependencies:
 
 ```sh
+cd imap-backup
 gem install bundler --version=2.3.22
 bundle install
 ```
 
-Check that it runs
+Check that it runs:
 
 ```sh
-bin/imap-backup help
+bin/imap-backup version
 ```
 
-Install dependencies
+If you get something like
 
-```sh
-cd imap-backup
-gem install bundler --version=2.3.22
-bundle install
 ```
+imap-backup 14.4.4
+```
+congratulations, you have succesfully built imap-backup.
```

Comment:
There should be a blank line after code blocks.

Reply:
Good catch!